pyfolio/utils.py

```diff
@@ -117,8 +118,9 @@ def get_symbol_rets(symbol):
             if datetime.now() - pd.to_datetime(
                     getmtime(filepath), unit='s') < pd.Timedelta(days=1):
                 rets = pd.read_hdf(filepath, 'df')
-        except:
-            pass
+        except Exception as e:
```

Comment:
Why are we swallowing the exception here?  Can we usefully recover from returning `None` as a benchmark?  If not, this is just going to create a confusing exception somewhere else that's further away from the source of the real problem.